modules/openapi-generator/src/main/resources/kotlin-spring/openapiDocumentationConfig.mustache

```diff
@@ -0,0 +1,68 @@
+package {{basePackage}}
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+import org.springframework.web.util.UriComponentsBuilder
+import springfox.documentation.builders.ApiInfoBuilder
+import springfox.documentation.builders.RequestHandlerSelectors
+import springfox.documentation.service.ApiInfo
+import springfox.documentation.service.Contact
+import springfox.documentation.spi.DocumentationType
+import springfox.documentation.spring.web.paths.Paths
+import springfox.documentation.spring.web.paths.RelativePathProvider
+import springfox.documentation.spring.web.plugins.Docket
+import springfox.documentation.swagger2.annotations.EnableSwagger2
+
+import javax.annotation.Generated
+import javax.servlet.ServletContext
+
+{{>generatedAnnotation}}
+@Configuration
+@EnableSwagger2
+class SpringFoxConfiguration {
+
+    fun apiInfo(): ApiInfo {
+        return ApiInfoBuilder()
+            .title("{{appName}}")
+            .description("{{{appDescription}}}")
+            .license("{{licenseInfo}}")
+            .licenseUrl("{{licenseUrl}}")
+            .termsOfServiceUrl("{{infoUrl}}")
+            .version("{{appVersion}}")
+            .contact(Contact("", "", "{{infoEmail}}"))
+            .build()
+    }
+
+    @Bean
+{{=<% %>=}}
+    fun customImplementation(servletContext: ServletContext, @Value("${openapi.<%title%>.base-path:<%>defaultBasePath%>}") basePath: String): Docket {
```

Comment:
I'm a little unsure about `"${openapi.<%title%>.base-path:<%>defaultBasePath%>}"`.

IntelliJ tells me that it is an unresolved reference, and `defaultBasePath` seems to be empty.